cmd/api-errors.go

```diff
@@ -287,6 +287,8 @@ const (
 	ErrAdminInvalidArgument
 	ErrAdminInvalidAccessKey
 	ErrAdminInvalidSecretKey
+	ErrAdminNoAccessKey
+	ErrAdminNoSecretKey
```

Comment:
```suggestion
	ErrAdminNoAccessKey
	ErrAdminNoSecretKey
```

Generally add it at the end to avoid too many changes

Reply:
End of that section of the list or end of the whole list?

Reply:
End of the whole list.